microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -0,0 +1,434 @@
+---
+title: Windowed vs. visual hosting of WebView2
+description: Deciding whether to have your app use windowed hosting vs. visual hosting of the WebView2 control.  Hosting WebView2 in windowed or visual environments.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 08/24/2022
```

Comment:
@victorthoang - are you able to update date & push a commit into this forked PR?

Reply:
Responded to you on Teams, but I was able to commit and tested by changing the date.